app_dart/lib/src/foundation/utils.dart

```diff
@@ -179,6 +179,20 @@ Future<List<Target>> getTargetsToRun(Iterable<Target> targets, List<String?> fil
   return targetsToRun;
 }
 
+/// Returns `true` if [ciYaml.postsubmitTargets] should be ran during presubmit.
+///
+/// This is an **experimental** feature that is only enabled for the
+/// `flutter/engine` repository if a specific label is present on the PR.
+bool includePostsubmitAsPresubmit(CiYaml ciYaml, PullRequest pullRequest) {
+  if (ciYaml.slug != Config.engineSlug) {
```

Comment:
You should be fine to remove the check. The label can be added when we're ready to test it out new repos.

Reply:
Done.

Reply:
Done.